src/plugins/integrations/HttpIntegration.ts

```diff
@@ -81,6 +82,10 @@ class HttpIntegration implements Integration {
           [HttpTags.HTTP_PATH]: path.split('?')[0],
           [HttpTags.HTTP_URL]: host + path,
           [HttpTags.QUERY_PARAMS]: queryParams,
+          [SpanTags.TOPOLOGY_VERTEX]: true,
+          [SpanTags.TRIGGER_DOMAIN_NAME]: LAMBDA_APPLICATION_DOMAIN_NAME,
+          [SpanTags.TRIGGER_CLASS_NAME]: LAMBDA_APPLICATION_CLASS_NAME,
+          [SpanTags.TRIGGER_OPERATION_NAMES]: [path.split('?')[0]],
```

Comment:
this must be `host + path` to be consistent with the span name.

Reply:
Done